src/serialize-util.ts

```diff
@@ -0,0 +1,166 @@
+import path from "path";
+import { SourceNode } from "source-map";
+import { FunctionlessNode } from "./node";
+
+export function undefinedExpr() {
+  return "undefined";
+}
+
```

Comment:
could you put all of the `serialize-*` into a folder? there is a lot here that doesn't apply elsewhere.